actionview/test/template/active_model_helper_test.rb

```diff
@@ -156,6 +159,13 @@ def test_hidden_field_does_not_render_errors
     )
   end
 
+  def test_text_field_with_association_errors
+    assert_dom_equal(
+      %(<div class="field_with_errors"><input type="text" name="mascot[company_id]" id="mascot_company_id" /></div>),
+      text_field("mascot", "company_id")
```

Comment:
Would this work with a `select` the same as it does with a `text_field`? Given the select use case is more likely is it worth adding a test for that?